test/util/display_SYSINFO_settings.py

```diff
@@ -1,31 +1,28 @@
-# Copyright 2016, EMC, Inc.
+'''
+Copyright 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
Not to hold up the approval of the PR, but just a note for PRs going forward.  When modifying the copyright in an existing file, the previous copyright should be preserved.  IOW  2016-1017.